renovate.json

```diff
@@ -321,6 +321,19 @@
     }
   ],
   "packageRules": [
+    {
+      "groupName": "Platform Autorisatie Beheer Component (PABC)",
+      "matchPackageNames": [
+        "ghcr.io/platform-autorisatie-beheer-component/pabc-migrations",
+        "ghcr.io/platform-autorisatie-beheer-component/pabc-api"
+      ],
+      "matchDatasources": [
+        "docker"
+      ],
+      "description": [
+        "Group PABC migrations and API updates into a single PR"
```

Comment:
Is that possible? Because they do have different digests. Who is going to set the digests then?